src/pages/admin/fees.tsx

```diff
@@ -26,14 +28,6 @@ function AdminFees() {
     return <UserNeedsMoreRights />;
   }
 
-  const transformFeeRate = (value: number | string): number => {
```

Comment:
why removing that ?

Reply:
The conversion is done by the sdk directly. User only needs to pass user friendly values to the sdk, and it will do the transformation internally